etc/config-template.toml

```diff
@@ -308,35 +310,209 @@ stats-dump-period-sec = 600
 # cache-index-and-filter-blocks = true
 # compaction-pri = 3
 
-# options for Column Family raft.
-# Column Family raft is used to store raft log and raft states.
-[rocksdb.raftcf]
-# compression-per-level = "no:no:lz4:lz4:lz4:zstd:zstd"
-# block-size = "64KB"
+[rocksdb.lockcf]
+# compression-per-level = "no:no:no:no:no:no:no"
+# block-size = "16KB"
 # write-buffer-size = "128MB"
 # max-write-buffer-number = 5
 # min-write-buffer-number-to-merge = 1
-# max-bytes-for-level-base = "512MB"
+# max-bytes-for-level-base = "128MB"
 # target-file-size-base = "32MB"
-
-# should tune to 256MB~2GB.
 # block-cache-size = "256MB"
-# level0-file-num-compaction-trigger = 4
+# level0-file-num-compaction-trigger = 1
 # level0-slowdown-writes-trigger = 20
 # level0-stop-writes-trigger = 36
 # cache-index-and-filter-blocks = true
 # compaction-pri = 0
 
-[rocksdb.lockcf]
-# compression-per-level = "no:no:no:no:no:no:no"
-# block-size = "16KB"
+[raft-rocksdb]
+# RocksDB info log directory, if set, RocksDB will save the log in `info-log-dir/LOG`.
+# info-log-dir = ""
+# Maximal size of the info log, if new info log will be created if size is large than it.
+# info-log-max-size = 0
+# Time for the info log file to roll (in seconds).
+# info-log-roll-time = "0s"
+
+# Maximum number of concurrent background jobs (compactions and flushes)
+# max-background-jobs = 6
```

Comment:
We must use different configs, what I mean here is to remove `max-background-jobs`.